README.md

```diff
@@ -1,9 +1,12 @@
-<img src="https://www.ga4gh.org/wp-content/themes/ga4gh-theme/gfx/GA-logo-horizontal-tag-RGB.svg" alt="GA4GH Logo" style="width: 400px;"/>
+<img src="https://www.ga4gh.org/wp-content/themes/ga4gh/dist/assets/svg/logos/logo-full-color.svg" alt="GA4GH Logo" style="width: 400px;"/>
+
+![release_badge](https://img.shields.io/github/v/tag/ga4gh/task-execution-schemas)
```

Comment:
I think it would be nice to add a URL target for the badge. In this case, I think it could bring us to the releases page: https://github.com/ga4gh/task-execution-schemas/releases

Reply:
Good idea. Updating.